runtime/gc_modron_startup/mminit.cpp

```diff
@@ -3054,7 +3054,14 @@ gcReinitializeDefaultsForRestore(J9VMThread* vmThread)
 	bool result = true;
 
 	PORT_ACCESS_FROM_JAVAVM(vm);
+	OMRPORT_ACCESS_FROM_J9PORT(PORTLIB);
 
+	extensions->usablePhysicalMemory = omrsysinfo_get_addressable_physical_memory();
+	uintptr_t candidateSoftMx = extensions->computeDefaultMaxHeapForJava(false);
```

Comment:
you can put a comment that Restore path is only active for post Java 8 releases, hence we always have false for enableOriginalJDK8HeapSizeCompatibilityOption